python/pfs/drp/stella/utils.py

```diff
@@ -1,5 +1,6 @@
 import lsst.afw.image as afwImage
 import numpy as np
+from pfs.datamodel.pfsFiberTrace import PfsFiberTrace as pFT
```

Comment:
Please don't use abbreviations like this; use the complete and correct class name